tc/core/cuda/cuda_rtc.cc

```diff
@@ -20,6 +20,7 @@
 #include <cuda_runtime.h>
 #include <nvrtc.h>
 
+#include "tc/core/check.h"
```

Comment:
Why only here (and a couple of other places)? Shouldn't every file that uses these calls have this include?